src/components/private-route/private-route.tsx

```diff
@@ -0,0 +1,20 @@
+import {Navigate} from 'react-router-dom';
+import {AppRoute, AuthorizationStatus} from '../../const.ts';
+
+type PrivateRouteProps = {
+    authorizationStatus: AuthorizationStatus;
+    elem: JSX.Element;
+}
+
+function PrivateRoute(props: PrivateRouteProps): JSX.Element {
```

Comment:
Лучше делать сразу деструктуризацию
```suggestion
function PrivateRoute({ authorizationStatus, elem }: PrivateRouteProps): JSX.Element {
```